src/posix/platform/openthread-posix-config.h

```diff
@@ -362,4 +362,15 @@
 #define OPENTHREAD_POSIX_CONFIG_PRODUCT_CONFIG_FILE "src/posix/platform/openthread.conf.example"
 #endif
 
+/**
+ * @def OPENTHREAD_POSIX_CONFIG_RCP_TIME_SYNC_INTERVAL
+ *
+ * This seeting configures the interval (in unit of microseconds) for host-rcp
+ * time sync. I.E., the host  will recalculate the time offset b/w host and RCP
+ * every interval.
+ *
+ */
+#ifndef OPENTHREAD_POSIX_CONFIG_RCP_TIME_SYNC_INTERVAL
+#define OPENTHREAD_POSIX_CONFIG_RCP_TIME_SYNC_INTERVAL (60 * US_PER_S)
```

Comment:
I think it is better to set the macro's value to (60 * 1000 * 1000). Because if a `.cpp` file only contain one header file `openthread-posix-config.h`, it may cause the compiler failed to find the definition of the `US_PER_S `.

Reply:
Updated.